cmd/ris-mirror/rismirror/router.go

```diff
@@ -16,6 +16,7 @@ type Router struct {
 	address     net.IP
 	vrfRegistry *vrf.VRFRegistry
 	vrfs        map[uint64]*_vrf
+	vrfClients  map[uint64][]*risclient.RISClient
```

Comment:
Given this changes at runtime and is accessed from multiple go routines, access needs to be synchronized or am I missing anything here? The same applies to the `vrfs` member I guess.

Reply:
I've added a mutex here